apps/hash-graph/lib/graph/src/api/rest.rs

```diff
@@ -130,18 +141,7 @@ pub fn rest_api_router<P: StorePool + Send + 'static>(
         .layer(Extension(dependencies.domain_regex))
         .layer(axum::middleware::from_fn(log_request_and_response))
         .layer(span_trace_layer())
-        .nest(
-            "/api-doc",
-            Router::new()
-                .route(
-                    "/openapi.json",
-                    get({
-                        let doc = open_api_doc;
-                        move || async { Json(doc) }
-                    }),
-                )
-                .route("/models/*path", get(serve_static_schema)),
-        )
+        .merge(openapi_only_router())
 }
 
 #[allow(
```

Comment:
Just a thing I noticed 🙃 

```suggestion
#[expect(
```

Reply:
[Change allow to expect](https://github.com/hashintel/hash/pull/2358/commits/3e455e857c5902f7355d2a79801876a223a4639e)

🙃

Reply:
Thanks for bearing with me 😂

Reply:
Always good to improve code where we can :)